### PR TITLE
VR-6125  Migrate verta.deployment to verta.endpoint

### PR DESCRIPTION
**Pull request by nhatsmrt**

### CODE REVIEW

File: client/verta/verta/endpoint/update/rules.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import abc
 import json
 
-from ...external import six
+from verta.external import six
```
Comment (convoliution) [inline client/verta/verta/endpoint/update/rules.py]: Was there a functional reason for this change? The relative import `...external` is strongly preferred.

**Review by convoliution — CHANGES_REQUESTED** [review]

File: client/verta/verta/endpoint/update/rules.py (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/verta/endpoint/update/rules.py]: I have no idea why this is changed 😱 
Need more tea

Comment (nhatsmrt) [reply client/verta/verta/endpoint/update/rules.py]: Hmm I think that's pycharm file moving weird shennanigans. Let me fix all of them.

Comment (nhatsmrt) [reply client/verta/verta/endpoint/update/rules.py]: Fixed.

**Review by convoliution — APPROVED** [review]

**Comment by convoliution** [comment]
fyi @evgeniilezhnin in case this impacts any of your PRs

**Pull request merged**